prep-for-module-3-frontend.markdown

```diff
@@ -10,11 +10,10 @@
 There are a bunch of JavaScript frameworks out there to use. At Turing, we choose to teach **React** - it's one of the most popular frameworks out there. Actually, a lot of developers will call React a library, which it is...but it can also be described as a framework. It's a little confusing, but the lines are blurred for React. Let's just say that React is a library with some rules and conventions to follow.
 
 Complete **one** of the tutorials below to get introduced to React:
-* [Tyler McGinnis's React Tutorial: A Comprehensive Guide to learning React.js in 2018](https://tylermcginnis.com/reactjs-tutorial-a-comprehensive-guide-to-building-apps-with-react/)
-* [React tutorial from the docs](https://reactjs.org/tutorial/tutorial.html)
-* [Original Scrimba React Course](https://scrimba.com/playlist/p7P5Hd)
+* [Tyler McGinnis's React Tutorial: A Comprehensive Guide to learning React.js in 2018](https://ui.dev/react-hooks)
```

Comment:
This one costs $149, I couldn't find any free trial or other free option.  I think we should remove it.

Reply:
I removed it for now until we find a better option